src/atomic64/mod.rs

```diff
@@ -53,6 +52,12 @@ mod test {
         au64.inc_by(1);
         assert_eq!(au64.get(), 1);
     }
+}
+
+#[cfg(all(test, feature = "bench"))]
+mod bench {
```

Comment:
Move it to benches

Reply:
it is not exposed so that cannot be benched outside the crate.